README.md

```diff
@@ -210,3 +211,23 @@ We are deliberately not using other project dependencies (notably `the-build`, `
 Solr support is not provided by default. It can be added later if needed.
 
 The `main` branch is locked against commits that are not in approved pull requests.
+
+
+## Services
+
+### Solr server
+
+A solr server is ready to be used at http://drupal-10-development.ddev.site:8983/solr/.
+
+`search_api_solr` server dependency is not currently installed, if you want to test the functionality try this branch [search_api_example](https://github.com/palantirnet/drupal-10-development/tree/search_api_example) or follow these steps.
+  * Install `search_api_solr` module `composer require drupal/search_api_solr`
```

Comment:
Should we do this via composer or via `ddev clone`? I suppose that using composer gets the module dependencies.

Reply:
@agentrickard I don't know how `ddev clone` works, so Im going to look for some information.